### PR TITLE
Jwks

### PR DESCRIPTION
**Pull request by libuweber**
# Pull Request Template

## Description
Support for JWKS Services in Microgateway 3.0

## Type of change

- [x] Feature (breaking change, which have impact to existing functionality)
- [x] Documentation

### CODE REVIEW

File: charts/microgateway/README.md.gotmpl [hunk 1]
```diff
@@ -230,11 +231,17 @@ Finally, apply the Helm chart configuration file with `-f` parameter.
 Please refer to the [Echo-Server Helm chart](https://artifacthub.io/packages/helm/ealenn/echo-server) to see all possible parameters of the Echo-Server Helm chart.
 
 ## DSL Configuration
-It is possible to use all Microgateway DSL configuration options within the 'dsl' configuration parameter.
+The Microgateway DSL configuration can be provided in 2 different ways: 
+- within the Helm Chart 'dsl' configuration parameter
+- in an existing ConfigMap mounted into the Microgatway pod
+
+:warning: **Changing the DSL configuration in a running system**:<br>
+The microgateway does not detect DSL changes at runtime. If the DSL configuration is managed by the Helm Chart, the Microgateway pod is restarted automatically after a DSL change.
```
Comment (orltom) [inline charts/microgateway/README.md.gotmpl]: ..the Microgateway pod is restarted...

helm will trigger a rollout deployment not a restart

File: charts/microgateway/templates/_helpers.tpl [hunk 2]
```diff
@@ -107,6 +107,21 @@ return true if license secret should be mounted
 {{- end -}}
 {{- end -}}
 
+{{/*
```
Comment (orltom) [inline charts/microgateway/templates/_helpers.tpl:110]: just out of curiosity: why does the filename start with a underscore?

Comment (libuweber) [reply charts/microgateway/templates/_helpers.tpl:110]: That's a Helm convention. Files starting with _ are assumed not to contain a manifest and will not be rendered to a K8s object definitions. The file is created when initializing a new Chart using `helm create my_chart` see https://helm.sh/docs/chart_template_guide/named_templates/ for more information.

**Review by orltom — APPROVED** [review]
Looks good to me

File: charts/microgateway/README.md.gotmpl (hunk 1 above)
Comment (libuweber) [reply charts/microgateway/README.md.gotmpl]: fixed

**Pull request merged**